repository/o.json

```diff
@@ -300,6 +300,17 @@
 				}
 			]
 		},
+		{
+			"name": "Ollama Integration",
+			"details": "https://github.com/dapepe/OllamaSublime",
+			"labels": ["AI", "Ollama", "LLM", "GPT"],
```

Comment:
This is really minor, but the convention is mostly for the labels to be lowercase. I don’t think they’re case sensitive, but you know, since we’re here I did want to mention that.